tests/update/patch_sle.pm

```diff
@@ -225,7 +225,8 @@ sub install_patterns {
             record_soft_failure 'bsc#1034541';
             next;
         }
-        zypper_call "in -t pattern $pt";
+        zypper_call("in -t pattern $pt", timeout => 900);
```

Comment:
No problem need increase the timeout value. I just have concern why need so much. By default it is 90s then change to 900s? I guess we can use a more suitable value, such as 180s. Too much timeout value will make user to wait too more time after a failure happened.

Reply:
> 
> 
> No problem need increase the timeout value. I just have concern why need so much. By default it is 90s then change to 900s? I guess we can use a more suitable value, such as 180s. Too much timeout value will make user to wait too more time after a failure happened.

default is 700 seconds, not 90s

Reply:
Ok, no problem.